src/Common/Constants.ts

```diff
@@ -17,4 +17,5 @@ export default class Constants {
   public static readonly transactionid = "transaction-id";
   public static readonly customerDisplayName = "customerDisplayName";
   public static readonly hiddenContentPlaceholder = "*content hidden*";
+  public static readonly validateAuthChatRecordTimeout = 5000;
```

Comment:
why is this a constant? Can it be configurable as well?

Reply:
I can make this configurable but made it a constant since validateAuthChatRecord is not exposed via interface
Assuming we do not want to expose this and keep it internal